lib/DDG/Goodie/DaysBetween.pm

```diff
@@ -13,19 +13,18 @@ zci is_cached => 0;
 zci answer_type => "days_between";
 
 my $datestring_regex = datestring_regex();
-my @monhs  = qw( Jan Feb Mar Apr May Jun Jul Aug Sep Oct Nov Dec );
+my @months  = qw( Jan Feb Mar Apr May Jun Jul Aug Sep Oct Nov Dec );
 
-handle query_lc => sub {
-    return unless ($_ =~ qr/(days since)(?: *)($datestring_regex)/i) || ($_ =~ qr/($datestring_regex) (?:(?:and|to) )?($datestring_regex)(?:[,]? inclusive)?$/i);
```

Comment:
@tagawa Using this regex will also pass queries like 'days between 12 July 2016', which makes no sense. I think we should compare that only queries like 'days since' should contain 1 date and others will have 2 dates.